Makefile

```diff
@@ -27,6 +27,7 @@ mandir ?= ${prefix}/share/man
 CFLAGS ?= -O2 -g -Wall -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2
 bashcompletiondir ?= ${datadir}/bash-completion/completions
 pkgconfigdatadir ?= $(datadir)/pkgconfig
+OPENSSL_LIBS ?= -DOPENSSL -lssl -lcrypto
```

Comment:
`configure` needs to error out if openssl is not present, like kmod:

https://github.com/dracutdevs/dracut/blob/master/configure#L58

It is guaranteed `OPENSSL_LIBS` will be set then.

Reply:
Perhaps only set the flag in configure if the library is installed to deal with "minimalist" distro's ?

Reply:
Doable with #ifdefery in dracut-install.c.